shared/teams/common/activity.tsx

```diff
@@ -5,7 +5,7 @@ import * as Types from '../../constants/types/teams'
 
 const activityToIcon: {[key in 'active' | 'recently']: Kb.IconType} = {
   active: 'iconfont-fire',
-  recently: 'iconfont-team-leave',
+  recently: 'iconfont-typing',
```

Comment:
are you sure you want `typing` and not `campfire-out`?

Reply:
https://app.zeplin.io/project/5e2b67989d77855729413dce/screen/5e2b61487119da7e69fb64a9

you tell me :D

Reply:
I think you want fire-out for this, and you want to use the typing icon in the place where you presumably already have it in the other screen